### DISCUSSION TITLE
Langfuse integration not working with LiteLLM

### DISCUSSION THREAD

**Discussion by med-dak** [open-webui/open-webui#5479] [Q&A]
I followed the documentation to configure Langfuse with LiteLLM, but it's not working as expected (https://docs.openwebui.com/tutorial/langfuse).

**Environment:**

- Open Web UI version: v0.3.10

- Langfuse version: 2.39.2

- Docker setup

**Expected behavior:** Open Web UI should connect to Langfuse and send data.

**Actual behavior:** No interaction or data is sent to Langfuse.

**Comment by flefevre** [comment]
Be careful that litellm is not anymore bundle with openwebui.
Personaly I use an external dockerised litellm that references the different llm served by ollama. I have configured litellm to reference an external dockerised langflow.
This configuration works.
